src/app/clients/page.tsx

```diff
@@ -15,10 +16,9 @@ const Clients: () => Promise<React.ReactElement> = async () => {
         <main>
             <h1> Clients Page </h1>
 
-            <p> Testing Supabase fetching </p>
-
             {/* This should be a separate component which is passed data via props */}
             <pre>{JSON.stringify(data, null, 4)}</pre>
+            <SampleDataViewerWithButton data={data![0]} />
```

Comment:
Can you remove this line? It shouldn't be on the final page (yet) - we're just making the component!